### ISSUE TITLE
Memory leak caused by repeated application of jQuery's not function

### ISSUE THREAD

**Issue by halter73** [mattbradley/livestampjs#19]
The offending code can be found here:

https://github.com/mattbradley/livestampjs/blob/1.1.2/livestamp.js#L62

Calling `livestampGlobal.update()` repeatedly causes unbounded growth of the `$livestamps` jQuery object. This happens because jQuery retains a reference to the previous selection in the return value of `not`.

A hacky workaround might be something like this:

```
$livestamps = $livestamps.not(toRemove);
delete $livestamps.prevObject;
```

I found this bug by investigating a memory leak in http://jabbr.net that causes the tab running JabbR to consume gigabytes of memory after running for days/weeks. This issue seems to be the primary cause.

**Comment by mattbradley** [comment]
Thanks for pointing this out. This issue has been brought up before and already fixed (#16), coincidently using the same solution you provided. The library on master (both livestamp.js and livestamp.min.js) has the updated code. I haven't gotten to releasing another version yet, so give master a try.

**Comment by halter73** [comment]
My bad. I definitely should have looked at master. Thanks.
